modules/renter.go

```diff
@@ -141,6 +141,8 @@ type HostDBEntry struct {
 	// ScanHistory is the set of scans performed on the host. It should always
 	// be ordered according to the scan's Timestamp, oldest to newest.
 	ScanHistory HostDBScans
```

Comment:
I know that this is not from this PR, but I think here is bad form, should just be []hdbs

The hdbss type is purely for sorting.